main.js

```diff
@@ -8,37 +8,40 @@ import { COLOR } from "./logger.js"
 import points from "./points.js";
 import { db, createTables } from "./sql.js";
 
-export const webhook = new WebhookClient({ url: config.webhook });
+export let webhook = null;
```

Comment:
can u pls rename it to errorhook? webhook could be anything

Reply:
done